x-pack/plugins/security_solution/public/management/pages/policy/store/policy_details/middleware.ts

```diff
@@ -43,23 +42,6 @@ export const policyDetailsMiddlewareFactory: ImmutableMiddlewareFactory<PolicyDe
         return;
       }
 
-      // Until we get the Default configuration into the Endpoint package so that the datasource has
```

Comment:
I moved this to the server side now that we have the ability to intercept the datasource create. This means that as soon as we create a datasource, the Endpoint policy will be there and we no longer have to navigate over to the Policy page and perform a "save" just so that the policy data is inserted into the DS.